File: 01_big_O_notation/05_rules/04_polinomial_rule/polinomial_rule.c
/*
f(n) is polynominal of dregree k -> f(n) = O(n^k)
*/

#include <stdio.h>
#include <stdlib.h>

int f(int n) {
  int accum = 0;

  for (int i = 0; i < n * n; i++) {
    accum += 1;
  }

  return accum;
}

int main() {
  printf("%d", f(5));
  return 0;
}

File: 01_big_O_notation/05_rules/04_polinomial_rule/polinomial_rule.js
/*
f(n) is polynominal of dregree k -> f(n) = O(n^k)
*/

function f(n) {
	let accum = 0

	for (let i = 0; i < n * n; i++) {
		accum += 1
	}

	return accum
}

n=5
console.log(f(n))
